test/jit/test_enum.py

```diff
@@ -108,15 +110,14 @@ def enum_comp(x: Foo) -> bool:
             scripted_enum_comp(Foo.ITEM1),
             False)
 
-
     def test_heterogenous_value_type_enum_error(self):
-        global Color
+        global ColorDiffType
```

Comment:
Do you still need to rename this now that you're calling `super().setUp()` and `super().tearDown()` correctly?

Reply:
Thanks for catching this. It is not needed, just a merge error.